README.md

```diff
@@ -1,2 +1,25 @@
 # Ruby-Tic-Tac-Toe
 Tic Tac Toe Game using OOP with ruby
+## Classical Tic-Tac-Toe in the terminal
+we have a 3X3 grid, with column 1-2-3 and rows a-b-c to choose which bracket we write the row a and the column 1
+so == a1 
+## what we had to do
+Think about how you would set up the different elements within the game… What should be a class? Instance variable? Method? A few minutes of thought can save you from wasting an hour of coding.
+
+Build your game, taking care to not share information between classes any more than you have to.
```

Comment:
- [ ] try to be a bit more cautious here, copy partying is ok but it should sound a bit more human, you could quote that as the rules for the project, give some context, etc.